cmd/terramate/cli/cli.go

```diff
@@ -383,21 +360,26 @@ func (c *cli) run() {
 			Msg("Handle `run <cmd>` command.")
 		c.runOnStacks()
 	case "generate":
-		logger.Debug().Msg("Handle `generate` command.")
-
 		report := generate.Do(c.root(), c.wd())
 		c.log(report.String())
 
 		if report.HasFailures() {
 			os.Exit(1)
 		}
-	case "metadata":
-		logger.Debug().
-			Msg("Handle `metadata` command.")
+	case "experimental globals":
+		c.printStacksGlobals()
+	case "experimental metadata":
 		c.printMetadata()
+	case "experimental init <paths>":
```

Comment:
`init` what? maybe change to `experimental init-stack` ?

Reply:
hmm sounds better indeed